llama_index/vector_stores/simple.py

```diff
@@ -107,8 +101,13 @@ def delete(self, doc_id: str, **delete_kwargs: Any) -> None:
     def query(
         self,
         query: VectorStoreQuery,
+        **kwargs: Any,
     ) -> VectorStoreQueryResult:
         """Get nodes for response."""
+        if query.filters is not None:
+            raise ValueError('Metadata filters not implemented for '
+                             "SimpleVectorStore yet.")
```

Comment:
Future PR? Is implementing filters for the simple index a pain? lol

Reply:
yeah probably fine to leave as future PR, makes "simple" vector store a bit less simple :)